### PR TITLE
Add annotation dataclass

### PR DESCRIPTION
**Pull request by dstansby**
Add an `Annotation` dataclass to make code more readable and less susceptible to errors (e.g. mixing up x/y when creating an annotation)

### CODE REVIEW

File: napari_arboretum/tree.py [hunk 1]
```diff
@@ -9,32 +9,41 @@
 WHITE = np.array([255, 255, 255, 255], dtype=np.uint8)
 RED = np.array([255, 0, 0, 255], dtype=np.uint8)
 
+# RGBA in range [0, 255]
+ColorType = Tuple[float, float, float, float]
```
Comment (paddyroddy) [inline napari_arboretum/tree.py:13]: I can never bring myself to do American spellings 😬

File: napari_arboretum/tree.py [hunk 2]
```diff
@@ -9,32 +9,41 @@
...
+
+
+@dataclass
+class Annotation:
+    x: float
+    y: float
+    text: str
```
Comment (paddyroddy) [inline napari_arboretum/tree.py:20]: I know before it was `tstr` which isn't great, but could this `text` be made something more descriptive? Even `label` would be better than `text` IMO as I'd at least know what it was for

**Review by paddyroddy — APPROVED** [review]
Wasn't sure if this was ready for review, but did it anyway (so won't merge)

File: napari_arboretum/tree.py (hunk 2 above)
Comment (dstansby) [reply napari_arboretum/tree.py:20]: 👍 `label` is a much better name, will update

**Pull request merged**